go/jsonclient/client_test.go

```diff
@@ -226,7 +234,9 @@ func TestPostAndParse(t *testing.T) {
 }
 
 func TestPostAndParseWithRetry(t *testing.T) {
-	leeway := time.Millisecond * 100
+	// leeway is quite large to accommodate for the RFC1123 Retry-After
+	// test
+	leeway := time.Millisecond * 750
```

Comment:
Why not make leeway another field in the `tests` structure, and preserve the more-accurate checking for the other cases?